### ISSUE TITLE
flashfocus not building:  pyyaml<6.0,>=5.1

### ISSUE THREAD

**Issue by seghers** [NixOS/nixpkgs#154342]
### Describe the bug
flashfocus won't build because pyyaml has a too recent version.
probably since pyyaml 6.0:
https://github.com/NixOS/nixpkgs/commit/db01e386f4fe725bfd1f733c26e74dc838156233#diff-97e5ee71df84aa882e708ef9f2ef0a816debc352c0569cdfdf759111004e2910

### Steps To Reproduce
1. nix-shell -p flashfocus

### Expected behavior
To build without errors
``` shell
ERROR: Could not find a version that satisfies the requirement pyyaml<6.0,>=5.1 (from flashfocus) (from versions: none)
ERROR: No matching distribution found for pyyaml<6.0,>=5.1
error: builder for '/nix/store/xxxxxxxxxxxxxx-flashfocus-2.2.3.drv' failed with exit code 1;
       last 10 log lines:
       > adding 'flashfocus-2.2.3.dist-info/top_level.txt'
       > adding 'flashfocus-2.2.3.dist-info/RECORD'
       > removing build/bdist.linux-x86_64/wheel
       > Finished executing setuptoolsBuildPhase
       > installing
       > Executing pipInstallPhase
       > /build/flashfocus-2.2.3/dist /build/flashfocus-2.2.3
       > Processing ./flashfocus-2.2.3-py3-none-any.whl
       > ERROR: Could not find a version that satisfies the requirement pyyaml<6.0,>=5.1 (from flashfocus) (from versions: none)
       > ERROR: No matching distribution found for pyyaml<6.0,>=5.1
       For full logs, run 'nix log /nix/store/xxxxxxxxxxx-flashfocus-2.2.3.drv'.
```

### Notify maintainers
@Artturin (flashfocus)
@dotlambda (pyyaml)

### Metadata
```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.12, NixOS, 22.05 (Quokka)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.6.0pre20211217_6e6e998`
 - nixpkgs: `/etc/channels/nixpkgs`
```

**Comment by dit7ya** [comment]
I am having a similar bug dependent on PyYaml.


```
error: builder for '/nix/store/1xqknnky3j7l44a2lqnimx18iayl74pp-python3.9-kaptan-0.5.12.drv' failed with exit code 1;
       last 10 log lines:
       > adding 'kaptan-0.5.12.dist-info/top_level.txt'
       > adding 'kaptan-0.5.12.dist-info/RECORD'
       > removing build/bdist.linux-x86_64/wheel
       > Finished executing setuptoolsBuildPhase
       > installing
       > Executing pipInstallPhase
       > /build/kaptan-0.5.12/dist /build/kaptan-0.5.12
       > Processing ./kaptan-0.5.12-py3-none-any.whl
       > ERROR: Could not find a version that satisfies the requirement PyYAML<6,>=3.13 (from kaptan) (from versions: none)
       > ERROR: No matching distribution found for PyYAML<6,>=3.13
       For full logs, run 'nix log /nix/store/1xqknnky3j7l44a2lqnimx18iayl74pp-python3.9-kaptan-0.5.12.drv'.
       ```

**Comment by dotlambda** [comment]
> I am having a similar bug dependent on PyYaml.

fixed by 17153b78f3491ccf176d6bc7de6ca7518cb62eb9
